Jenkinsfile

```diff
@@ -4,6 +4,10 @@ node('linux && docker') {
   def isBump = !!tag
   def isMaster = env.BRANCH_NAME == 'master'
 
+  if (!isMaster) {
+    return
+  }
+
```

Comment:
why modify Jenkins, is it not dead yet?

Reply:
it's not dead yet, it will be in the next PR.

Reply:
GOOD! 😈